_rules/link-non-empty-accessible-name-c487ae.md

```diff
@@ -85,7 +85,7 @@ This `a` element has an [accessible name][] from its content.
 This `div` element has an [explicit semantic role](#explicit-role) of `link` and an [accessible name][] from its content.
 
 ```html
-<div role="link" onclick="window.location.href='https://www.w3.org/WAI/'">Web Accessibility Initiative (WAI)</div>
+<div role="link" onclick="window.location.href='https://www.w3.org/WAI/'" tabindex="0">Web Accessibility Initiative (WAI)</div>
```

Comment:
We should add a keyboard event here as well, otherwise even with the tabindex you cannot activate the link.

Reply:
https://github.com/act-rules/act-rules.github.io/blob/develop/test-assets/bypass-blocks-cf77f2/click-on-enter.js (or a stripped down version) should do the trick.

Reply:
Huh! I did not know that! Thank you.